fs/find-packages/test/index.ts

```diff
@@ -59,6 +59,20 @@ test('ignore packages by patterns', async () => {
   expect([pkgs[0].manifest.name, pkgs[1].manifest.name].sort(compare)).toStrictEqual(['component-1', 'component-2'])
 })
 
+test('ignore packages by patterns with starts with !/', async () => {
+  const root = path.join(fixtures, 'many-pkgs')
+  const pkgs = await findPackages(root, { patterns: ['**', '!/libs/**'] })
```

Comment:
I verified that if the pattern starts with `!/`, both fast-glob and tinyglobby seem to ignore this value. Therefore, the returned result should be reasonable.

Reply:
In this case, tinyglobby and fast-glob do not handle the same results on Windows.